pycfmodel/model/cf_model.py

```diff
@@ -30,11 +31,14 @@ class CFModel(CustomModel):
     - Transform: For serverless applications, specifies the version of the AWS Serverless Application Model (AWS SAM) to use.
 
     More info at [AWS Docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/template-anatomy.html)
+
+    More info for Globals at [AWS Globals Docs](https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/sam-specification-template-anatomy-globals.html)
     """
 
     AWSTemplateFormatVersion: Optional[date]
     Conditions: Optional[Dict] = {}
     Description: Optional[str] = None
+    Globals: Optional[Dict] = {}
```

Comment:
I'd create a subclass for SAM. Normal CF does not support Globals

Reply:
It was one of the ideas I had in mind. I'd try to give this PR some development time this week and do a subclass instead for SAM 👍